disperser/batcher/grpc/dispatcher.go

```diff
@@ -132,7 +132,7 @@ func (c *dispatcher) sendChunks(ctx context.Context, blobs []*core.BlobMessage,
 	if err != nil {
 		return nil, err
 	}
-	c.logger.Debug("sending chunks to operator", "operator", op.Socket, "num blobs", len(blobs), "size", totalSize, "request message size", proto.Size(request), "request serialization time", time.Since(start))
+	c.logger.Debug("sending chunks to operator", "operator", op.Socket, "num blobs", len(blobs), "size", totalSize, "request message size", proto.Size(request), "request serialization time", time.Since(start), "use Gnark chunk encoding", c.EnableGnarkBundleEncoding)
```

Comment:
nit: It would be nicer to keep the labels one word without spaces (`numBlobs`, `requestMessageSize`, etc), to support label filter queries. I think LogQL queries bork if labels contain spaces